Classes/DDASLLogger.m

```diff
@@ -117,7 +119,7 @@ - (void)logMessage:(DDLogMessage *)logMessage {
 }
 
 - (NSString *)loggerName {
```

Comment:
`loggerName` should now be of type `DDLoggerName`. This also needs to be changed in the `DDLogger` protocol.

Reply:
@ffried 
And where in the `DDLog.h` this file `DDLoggerNames.h` should be imported? It is far enough between protocol and import section if it is added to the head.

Reply:
I suggest to add it right above `@protocol DDLogger` definition.

Reply:
I'd just import it below `#import "DDLegacyMacros.h"`. Since it only contains constants, it's fine.

Reply:
> I suggest to add it right above @protocol DDLogger definition.

Please keep the imports at the top of the file. Otherwise everyone needs to search the whole file for imports.